### DISCUSSION TITLE
Copilot and RStudio

### DISCUSSION THREAD

**Discussion by dfeistauer** [community/community#127398] [GitHub Education]
### Select Topic Area

Question

### Body

Hi,

I would like to instruct multiple students on how to use CoPilot in RStudio. We have all applied for the education plan and have been granted approval.

As a teacher, I was able to successfully connect CoPilot in RStudio. However, when my students sign in, they encounter an additional "activate" button which directs them to this webpage: 
<img width="1273" alt="image" src="https://github.com/community/community/assets/44641119/fa7a273f-d7fd-4761-9172-42e2be3b0310">

What do they or I have to do to use the copilot?

cheers
Daniela

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @dfeistauer ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by dfeistauer** [reply]
thanks.

**Comment by dfeistauer** [comment]
Solution: you need to press start free trial.
